src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -151,8 +151,8 @@
     <PackageReference Include="DotNetZip" Version="1.13.3" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.0.0-preview.3.16281" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.0.0-preview.4.16394" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
The package `https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson/6.0.0-rc.1.21452.15`, which WebHost uses, moved forward to 13.0.1, which required the CLI to also move forward.